.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+name: Release OpenAPI spec
+on:
+  repository_dispatch:
+    types: [release-openapi]
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        required: true
+        type: string
+        description: The version to be released
+
+env:
+  git_user_name: localstack[bot]
+  git_user_email: localstack-bot@users.noreply.github.com
+
+jobs:
+  release-localstack-openapi:
+    runs-on: buildjet-2vcpu-ubuntu-2204
+    env:
+      release: ${{ github.event_name == 'workflow_dispatch' && inputs.releaseVersion || github.event.client_payload.version}}
```

Comment:
nit: Would it make sense to have the same `client_payload` input name as defined in the `workflow_dispatch`? And is the `client_payload` maybe even already part of the inputs context? In this case you would be able to really just directly use it? :)